PluginController/EntityPluginController.php

```diff
@@ -397,4 +389,10 @@ protected function doGetPaymentInstruction($id)
 
         return $paymentInstruction;
     }
+
+    protected function doRollback()
+    {
+        $this->entityManager->getConnection()->rollback();
+        $this->entityManager->close();
```

Comment:
I tend to agree with @makasim, this is a good opportunity to fix this. We should just do `rollback` and not `close`.